lib/pltr/v2/helpers/cards.js

```diff
@@ -24,8 +24,13 @@ export function cardMapping(sortedBeats, sortedLines, card2Dmap, currentLine) {
 }
 
 function lineIsHidden(line, currentLine) {
-  if (!currentLine) return false
-  return line.id != currentLine
+  // if no outlinefilter return all
+  if (Array.isArray(currentLine) && !currentLine.length) return false
```

Comment:
This line is redundant.  If it's an array then we don't need to check whether it has a length to run `.includes` on it.